packages/p2p/src/pages/my-profile/my-profile-stats/my-profile-name/my-profile-name-business-hours/my-profile-name-business-hours.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Icon, Text, Tooltip } from '@deriv/components';
+import { Localize, localize } from 'Components/i18next';
+import { getTextSize } from 'Utils/responsive';
+import './my-profile-name-business-hours.scss';
+
+type TMyProfileBusinessHoursProps = {
+    onClickMessage: () => void;
+};
+
+const MyProfileNameBusinessHours = ({ onClickMessage }: TMyProfileBusinessHoursProps) => {
+    return (
+        <div className='my-profile-name-business-hours'>
+            <Tooltip
+                alignment='top'
+                className='my-profile-name-business-hours__tooltip'
+                message={localize('Business hour')}
+                onClickMessage={onClickMessage}
```

Comment:
Why do you define the onclick for the tooltip instead of the actual element?

Reply:
it was how they wanted it in design 😓 
![Screenshot 2024-05-27 at 5 15 31 PM](https://github.com/binary-com/deriv-app/assets/103412909/4e9f11db-d3c5-44e0-ad20-eb02ccd0c9f2)

Reply:
I don't think it is the best user experience, even in the video that you shared, you're trying it multiple times to click on the label. I believe the onclick should be on the "Open" label. Can you check with the UX team.